test/db/tools/rz_asm

```diff
@@ -395,6 +395,7 @@ EOF
 RUN
 
 NAME=rz-asm -A with -c and -o
+BROKEN=1
```

Comment:
Why this one got broken?

Reply:
Because it assumes that all `MSR` instructions sets the flags in the `cpsr` register. Which is just not true. Some set flags in the `apsr`/`spsr` or simply move a value to another system register.

The `esil` instruction does not even remotely handle these cases.
Should I open an issue about it?

Reply:
Please do, yes